spec/support/flow_helpers.rb

```diff
@@ -125,6 +125,22 @@ def fill_in_property_entry_screen(screen_name: :property_entry, form_name: :clie
   click_on "Save and continue"
 end
 
+def fill_in_housing_costs_screen(screen_name: :housing_costs, form_name: :housing_costs)
```

Comment:
The `screen_name` and `form_name` variables are redundant - again, it's a pattern that only gets used when we reuse the same method for both client and partner versions of a screen. So you can delete them both, and just hard-code `confirm_screen :housing_costs`, `fill_in "housing_costs_form[housing_payments]", with: "0"` etc, because there won't be a scenario where we want to use a _different_ screen name or form name here.